data/al/legislature/Adline-Clarke-4cb1aeb5-d822-4f2c-a7e6-ae2a132acf17.yml

```diff
@@ -4,7 +4,6 @@ given_name: Adline Cecelia
 family_name: Clarke
 gender: Female
 email: adline.clarke@alhouse.gov
-image: https://storage.googleapis.com/enview-public-general/legislative-people-photos/51335.jpg
```

Comment:
I see you merged in the previous PR here. So we lost the source links. We probably need to revert these to the old source links.